events.js:287
      throw er; // Unhandled 'error' event
      ^

Error: Connection lost: The server closed the connection.
    at Protocol.end (D:\business_licence_db\project\server\node_modules\mysql\lib\protocol\Protocol.js:112:13)
    at Socket.<anonymous> (D:\business_licence_db\project\server\node_modules\mysql\lib\Connection.js:97:28)
    at Socket.<anonymous> (D:\business_licence_db\project\server\node_modules\mysql\lib\Connection.js:525:10)
    at Socket.emit (events.js:322:22)
    at endReadableNT (_stream_readable.js:1187:12)
    at processTicksAndRejections (internal/process/task_queues.js:84:21)
Emitted 'error' event on Connection instance at:
    at Connection._handleProtocolError (D:\business_licence_db\project\server\node_modules\mysql\lib\Connection.js:426:8)
    at Protocol.emit (events.js:310:20)
    at Protocol._delegateError (D:\business_licence_db\project\server\node_modules\mysql\lib\protocol\Protocol.js:398:10)
    at Protocol.end (D:\business_licence_db\project\server\node_modules\mysql\lib\protocol\Protocol.js:116:8)
    at Socket.<anonymous> (D:\business_licence_db\project\server\node_modules\mysql\lib\Connection.js:97:28)
    [... lines matching original stack trace ...]
    at processTicksAndRejections (internal/process/task_queues.js:84:21) {
  fatal: true,
  code: 'PROTOCOL_CONNECTION_LOST'
}
(node:3976) UnhandledPromiseRejectionWarning: TypeError: Cannot read property '0' of undefined
    at D:\business_licence_db\project\server\Service\webService.js:523:29
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:3976) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:3976) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
